docs/production/multiple-organizations.md

```diff
@@ -118,12 +118,7 @@ avoid confusion as to why there's an extra realm when inspecting the
 Zulip database.
 
 Every Zulip server comes with 1 realm that isn't created by users: the
-`zulipinternal` realm.  By default, this realm only contains the Zulip "system
-bots".  You can get a list of these on your system via
-`./scripts/get-django-setting INTERNAL_BOTS`, but this is where bots
-like "Notification Bot", "Welcome Bot", etc. exist.  In the future,
-we're considering moving these bots to exist in every realm, so that
-we wouldn't need the system realm anymore.
+`zulipinternal` realm.  TODO: What should we write now that cross-realm bots are gone?
```

Comment:
What do we want to write here? The realm still has some use - some notifications get posted to its streams, e.g this:
```
    # We also send a notification to the Zulip administrative realm
    admin_realm = get_realm(settings.SYSTEM_BOT_REALM)
    admin_realm_sender = get_system_bot(sender_email, admin_realm.id)
    try:
        # Check whether the stream exists
        signups_stream = get_signups_stream(admin_realm)
        # We intentionally use the same strings as above to avoid translation burden.
        message = _("{user} just signed up for Zulip. (total: {user_count})").format(
            user=f"{user_profile.full_name} <`{user_profile.email}`>", user_count=user_count
        )
        internal_send_stream_message(
            admin_realm_sender, signups_stream, user_profile.realm.display_subdomain, message
        )

    except Stream.DoesNotExist:
        # If the signups stream hasn't been created in the admin
        # realm, don't auto-create it to send to it; just do nothing.
        pass
```

Reply:
I think probably the right model is to have the `SYSTEM_BOT_REALM` setting control which realm is used for that, but have the default be that it's unset and we don't attempt to send those notifications.